exercises/parallel-letter-frequency/src/test/java/ParallelLetterFrequencyTest.java

```diff
@@ -0,0 +1,216 @@
+import org.junit.Test;
+import org.junit.Ignore;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
+
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ForkJoinPool;
+import java.util.stream.Collectors;
+import java.util.stream.LongStream;
+import java.util.*;
+
+public class ParallelLetterFrequencyTest {
+
+    // Poem by Friedrich Schiller. The corresponding music is the European Anthem.
+    private String OdeAnDieFreude =
+            "Freude schöner Götterfunken\n" +
+                    "Tochter aus Elysium,\n" +
+                    "Wir betreten feuertrunken,\n" +
+                    "Himmlische, dein Heiligtum!\n" +
+                    "Deine Zauber binden wieder\n" +
+                    "Was die Mode streng geteilt;\n" +
+                    "Alle Menschen werden Brüder,\n" +
+                    "Wo dein sanfter Flügel weilt.";
+
+    // Dutch national anthem
+    private String Wilhelmus =
+            "Wilhelmus van Nassouwe\n" +
+                    "ben ik, van Duitsen bloed,\n" +
+                    "den vaderland getrouwe\n" +
+                    "blijf ik tot in den dood.\n" +
+                    "Een Prinse van Oranje\n" +
+                    "ben ik, vrij, onverveerd,\n" +
+                    "den Koning van Hispanje\n" +
+                    "heb ik altijd geëerd.";
+
+    // American national anthem
+    private String StarSpangledBanner =
+            "O say can you see by the dawn's early light,\n" +
+                    "What so proudly we hailed at the twilight's last gleaming,\n" +
+                    "Whose broad stripes and bright stars through the perilous fight,\n" +
+                    "O'er the ramparts we watched, were so gallantly streaming?\n" +
+                    "And the rockets' red glare, the bombs bursting in air,\n" +
+                    "Gave proof through the night that our flag was still there;\n" +
+                    "O say does that star-spangled banner yet wave,\n" +
+                    "O'er the land of the free and the home of the brave?\n";
+
+
+    @Test
+    public void noTextsMeansNoLetters() {
+        String input = "";
+        Map<Integer, Integer> expectedOutput = new HashMap<Integer, Integer>();
+        ParallelLetterFrequency p = new ParallelLetterFrequency(input);
+
+        assertEquals(expectedOutput, p.letterCounts());
+    }
+
+    @Test
+    @Ignore
```

Comment:
This should be `Ignore("Remove to run test")` as explained in the [POLICY doc](https://github.com/exercism/java/blob/master/POLICIES.md#ignore-noninitial-tests) :)

Reply:
`@Ignore(...)`s should also be placed above the `@Test` annotation, per the POLICIES doc!